docker-hub/release-notes.md

```diff
@@ -11,6 +11,11 @@ known issues for each Docker Hub release.
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## 2022-09-12
+
+### Enhancement
+
+You can now [export a CSV file of users](../docker-hub/members.md/#export-members) from organizations that you own.
```

Comment:
users or members?

Reply:
Thanks! It should be members. Fixed in the following commits.